src/main/io/ledstrip.c

```diff
@@ -938,6 +938,30 @@ static void applyLarsonScannerLayer(bool updateNow, timeUs_t *timer)
     }
 }
 
+static void applyRainbowLayer(bool updateNow, timeUs_t *timer)
+{
+    int delta = 24;
```

Comment:
Demands a more close IMO. Also since we use C, `define` prepossessor constants are preferred